rpa-manager/modules/ROOT/pages/processautomation-manage.adoc

```diff
@@ -109,7 +116,7 @@ Change the manager of a project to represent changed responsibilities:
 . Click on the name of the category of the project.
 . In the *<category name>* view, click the *New Project Manager* icon image:newprojectmanager-icon.png[user-with-tie symbol,1.5%,1.5%] on the panel of the project for which you want to change the project manager.
 . In the window *New project manager*, select a user.
-. If you do not want to include the previous manager into the project team, unselect the checkbox *Add the current Project Manager to the project team. The current Project Manager will be added with no assigned phases and thus only able to see this project.*
+. If you do not want to include the previous manager into the project team, unselect the checkbox *Add the current Project Manager to the project team. The current Project Manager is added with no assigned phases and thus only able to see this project.*
```

Comment:
```suggestion
. If you do not want to include the previous manager in the project team, unselect the checkbox *Add the current Project Manager to the project team. The current Project Manager is added with no assigned phases and thus only able to see this project.*
```

Reply:
Is it correct to have all of the text above bolded or can you  write this instead? *Add the current Project Manager to the project team*.

Reply:
All the bolded text is UI text for the checkbox.